contrib/sssd.spec.in

```diff
@@ -296,6 +298,9 @@ Requires: python3-sssdconfig = %{version}-%{release}
 Requires: python2-sss = %{version}-%{release}
 Requires: python2-sssdconfig = %{version}-%{release}
 %endif
+%if (0%{?use_systemd} == 0)
+Requires: initscripts
+%endif
```

Comment:
It would be better to require /sbin/service as other packages
```
sh# rpm -e initscripts
error: Failed dependencies:
        initscripts >= 8.83-1 is needed by (installed) plymouth-0.8.3-27.el6.centos.1.x86_64
        /sbin/service is needed by (installed) policycoreutils-2.0.83-29.el6.x86_64
        /sbin/service is needed by (installed) iputils-20071127-21.el6.x86_64
```